uasc/secure_channel.go

```diff
@@ -262,6 +415,9 @@ func (s *SecureChannel) recv() {
 			if err == io.EOF {
 				return
 			}
+			if err != nil {
+				log.Fatal(err)
```

Comment:
`log.Fatal` might be a problem here.

Reply:
Yep; remnant of debugging that I forgot to remove.  This will disappear.